de.ient.RDPlot.yaml

```diff
@@ -0,0 +1,162 @@
+app-id: de.ient.RDPlot
```

Comment:
Please follow this style guide: https://github.com/flathub/flathub/wiki/YAML-Style-Guide